bitcanna/docker-compose.yml

```diff
@@ -2,17 +2,16 @@ version: '3.4'
 
 services:
   node_1:
-    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.26-bitcanna-v1.5.3
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.26-generic
     ports:
       - '26656:26656'
       - '26657:26657'
       - '1317:1317'
     environment:
-      - MONIKER=node_1
+      - BINARY_URL=https://github.com/BitCannaGlobal/bcna/releases/download/v1.6.2/bcna_linux_amd64.tar.gz
+      - MONIKER=R-Drone
       - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/bitcanna/chain.json
       - P2P_POLKACHU=1
       - STATESYNC_POLKACHU=1
-    env_file:
-      - ../.env
```

Comment:
please keep this. The rest of the projects use that for specifying the S3 creds.